### ISSUE TITLE
Why did my voxel_loss_coarse become nan when training

### ISSUE THREAD

**Issue by passingdragon** [alibaba-damo-academy/former3d#8]

**Comment by passingdragon** [comment]
I set sync_norm = false

**Comment by weihaosky** [comment]
I have not met this problem. Do you use the same training setting as stated in Readme? Including the batch size and gpus

**Comment by passingdragon** [comment]
I set n_images_train = 10,use one RTX 3090.Could it be caused by too few Gpus？

**Comment by weihaosky** [comment]
Yes. Reducing n_images_train and the number of GPUs would reduce the batch size, which may cause the training unstable. I suggest you first try to use n_images_train=20.
